galsim/wfirst/wfirst_detectors.py

```diff
@@ -96,7 +96,11 @@ def applyPersistence(img, prev_exposures):
     @param prev_exposures    List of up to 8 Image instances in the order of exposures, with the
                              recent exposure being the first element.
     """
-    img.applyPersistence(img_list,wfirst.persistence_coefficients)
+    if hasattr(prev_exposures,'__iter__'):
+      n_exp = min(len(prev_exposures),len(galsim.wfirst.persistence_coefficients))
+      img.applyPersistence(prev_exposures[:n_exp],galsim.wfirst.persistence_coefficients[:n_exp])
```

Comment:
This step is necessary only because the `applyPersistence` routine insists that the length of the list of Image instances and coefficients be the same. The program does not really crash if they differed; an error is raised in the suspicion that the user might have been wrong. We could change that to simple a warning which can then simply the `wfirst.applyPersistence` routine.